docs/output.md

```diff
@@ -11,6 +11,13 @@ This document describes the output produced by the pipeline. Most of the plots a
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
 <!-- TODO nf-core: Write this documentation describing your workflow's output -->
+Reading this you likely have already experienced first hand that single-cell sequencing data is big, exceeding the amount of RAM available to most computers. Nonetheless, algorithms may demand expression values of a particular set of genes for a particular set of cells at any time.
```

Comment:
```suggestion
Reading this you likely have already experienced first hand that single-cell sequencing data is big, exceeding the amount of RAM available to most computers. Nonetheless, algorithms may demand expression values of a particular set of genes for a particular set of cells at any time.
```
Reading this, you likely have already experienced firsthand that single-cell sequencing data produces enormous zero-inflated count matrices that hardly fit into the memory of any computer if processed as a dense representation. For this reason, single-cell data is mainly processed as sparse matrices.